src/index.js

```diff
@@ -382,125 +506,10 @@ const renderDialButtons = () => {
         }
 
         dialButtonPadElement.appendChild(dialButtonElement)
+        renderFooter()
```

Comment:
At this point of time, you are inside .forEach loop, so `renderFooter` is called n times - this is unnecessary.